data/README.md

```diff
@@ -1,8 +1,10 @@
 # Reverb Data Engineering Coding Challenge
-This technical challenge is focused on the issues faced by data engineers at Reverb. There is no one correct approach - the task outline below is meant to start a conversation about domain modeling and tradeoffs.
+
+This technical challenge is focused on the issues faced by data engineers at Reverb. There is no one correct approach -- the task outline below is meant to start a conversation about domain modeling and tradeoffs.
```

Comment:
maybe pedantic, but should probably use an em dash `—` instead of a double-dash.